.gitlab-ci.yml

```diff
@@ -624,12 +616,12 @@ release:distribution:
     - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ./.npmrc
     - echo 'Publishing library'
     - >
-      nix-shell --run $'
+      nix-shell --arg ci true --run $'
       npm publish --access public;
       '
     - echo 'Releasing application builds'
     - >
-      nix-shell --run $'
+      nix-shell --arg ci true --run $'
       gh release \
```

Comment:
Is the indentation here correct?

Reply:
It looks off on my mobile screen.

Reply:
I didn't change the indenting there so I think it's correct